pkgs/development/ocaml-modules/angstrom/default.nix

```diff
@@ -1,37 +1,25 @@
-{ stdenv, fetchFromGitHub, ocaml, findlib, dune, alcotest, result
```

Comment:
I don’t like this change: there is no need for recursion (the package described here is an attribute of its argument…) and it seems to make overriding more difficult.

Reply:
Yes, that's an issue. I was just seeing how much the derivation could be simplified, but of course, there is no need to go that far.